.github/workflows/test-python.yaml

```diff
@@ -2,43 +2,42 @@ name: Test Python
 on:
   workflow_call:
     inputs:
-      unit-platforms:
+      fast-platforms:
         required: false
         type: string
-        default: '["ubuntu-22.04", "ubuntu-24.04", "windows-latest", "macos-latest"]'
+        default: '["jammy", "noble"]'
```

Comment:
don't we need to further restrict these to `amd64`?

Reply:
We can and we needn't. I added the architectures in the self-test in order to check that we could add lists of tags, but I figured here it would be a good default to simply run the tests on the first available runner regardless of architecture.